augurpy/read_load.py

```diff
@@ -30,26 +30,17 @@ def load(
         out = input
 
     elif isinstance(input, DataFrame):
-        try:
-            cell_type = input[cell_type_col]
-            label = input[label_col]
-        except KeyError:
-            print("[bold yellow]No column names matching cell_type_col and label_col. Looking in meta data.")
+        if meta is None:
             try:
-                cell_type = meta[cell_type_col]
-                label = meta[label_col]
-            except (KeyError, TypeError):
-                print("Missing label and / or cell type column.")
-                raise
+                cell_type = input[cell_type_col]
+                label = input[label_col]
+            except KeyError:
+                print("[bold red]No column names matching cell_type_col and label_col.")
 
-            else:
-                out = AnnData(X=input, obs={"cell_type": cell_type, "label": label})
+        label = input[label_col] if meta is None else meta[label_col]
+        cell_type = input[cell_type_col] if meta is None else meta[cell_type_col]
 
-        else:
-            out = AnnData(X=input, obs={"cell_type": cell_type, "label": label})
-
-    else:
-        raise Exception("Not valid input.")
+        out = AnnData(X=input, obs={"cell_type": cell_type, "label": label})
```

Comment:
Not really happy with the name "out". Can we find something better?

Reply:
How about adata? Changed it to adata